### DISCUSSION TITLE
Use custom icons for collection.

### DISCUSSION THREAD

**Discussion by dkkb** [outline/outline#3208] [Show and tell]
<img width="245" src="https://user-images.githubusercontent.com/82504881/156997057-2789ea76-2dad-4129-98d0-aabcd09cf6ba.png">

😀

**Comment by Limezy** [comment]
Would you mind sharing how you did ?

**Reply by begemotik** [reply]
@dkkb would appreciate sharing as well

**Reply by dkkb** [reply]
Sure, https://github.com/dkkb/outline/commit/9f874823ba544782bd54bf3c101bc9ec9e5697a0.

It's a tricky way to achieve custom icons. The original implementation is to create svg element for icons, so I tried to find some svg icons and copy them into my code. It's not easy, because I like colorful icons andI tried to look up open source icon library (like font-awesome or other else) to integrate into Outline but it didn't work. 

Here're two small issues with my implementation:
1. The icons for collections only support single color, and I think it makes it easy to fit dark mode and light mode, when I use Rust-lang icon which is black, it's not perfect in dark mode. (But I'm ok with it, maybe it's not perfect for production)
2. The size of icons, I forgot the details about it, when I create new icons with `const view = rest.view ? rest.view : "0 0 48 48";` I need to adjust some CSS style.

Sorry for the delay.

@begemotik @Limezy

**Reply by dmnkdefault** [reply]
Hey there, is there any way you could share it again, the link doesnt seem to work anymore. I would appreciate it

**Comment by Novapixel1010** [comment]
This would be really cool. But would need to be compatible with dark and light mode.
